### PR TITLE
Add max_speed to iso_importer

### PR DESCRIPTION
**Pull request by sean797**

### CODE REVIEW

File: templates/iso_importer.json.erb [hunk 1]
```diff
@@ -19,4 +19,9 @@
     <% else %>
     "proxy_password": null
```
Comment (ekohl) [inline templates/iso_importer.json.erb]: This needs a trailing comma now

File: templates/iso_importer.json.erb [hunk 2]
```diff
@@ -19,4 +19,9 @@
...
     <% end %>
+    <% unless [nil, :undefined, :undef, ''].include?(scope['pulp::real_yum_max_speed']) -%>
+    "max_speed": <%= scope['pulp::real_yum_max_speed'] %>,
```
Comment (ekohl) [inline templates/iso_importer.json.erb]: Doesn't need a trailing comma. Same below

**Comment by ekohl** [comment]
I'm wondering whether we should use https://github.com/puppetlabs/puppetlabs-stdlib#to_json_pretty for these files.

**Comment by sean797** [comment]
Thanks for the quick review.. +1 to to_json_pretty in another PR

**Comment by ekohl** [comment]
I do wonder if it's obvious that the `yum_max_speed` parameter also counts for the ISO importer.

**Comment by sean797** [comment]
I wouldn't say its clear that it counts for RPM either ;)

Updated the doc  :+1:

**Comment by sean797** [comment]
Ugh, just realised what you were saying.. I'm not too keen on having `iso_max_speed` as well, and renaming it is a pain :cry:

**Comment by ehelms** [comment]
Given these live in the same plugin, I am OK with this one parameter for now.

**Pull request merged**